src/applications/facility-locator/components/markers/FacilityMarker.jsx

```diff
@@ -0,0 +1,25 @@
+import DivMarker from './DivMarker';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+
+class FacilityMarker extends Component {
+  render() {
+    const { position, onClick, markerText } = this.props;
+    return (
+      <DivMarker position={position} onClick={onClick}>
+        <span className="i-pin-map"> {markerText}</span>
+      </DivMarker>
+    );
+  }
+}
+
+FacilityMarker.defaultProps = {
+  onClick: () => {},
+};
```

Comment:
I like this so you don't have to worry about calling the function with a prop, but is there a case where you expect that prop to not exist?